ClinGen/clincoded
# Connects to #737. Connects to #788. Calculate CI and standardize decimal places on pop tab

Instance: http://737-mc-calculate-ci-2374fe1-minchoi.instance.clinicalgenome.org/

#### Connects to #737:

* CI fields only show up when interpretation is added
* CI lower and upper values update when desired CI is changed
* Clearing the the desired CI field or putting in bad characters will default the value to 95 on clicking away
* Desired CI values are saved on evaluation form saving, and is pre-filled with saved value on re-loading the interpretation

![image](https://cloud.githubusercontent.com/assets/4326866/16751326/60fdb886-478d-11e6-816f-e2d60dba3536.png)

*Testing:*

1. Get to VCI hub, populations tab
2. Add interpretation and confirm that the CI fields now show up, with 95 as default value
3. Change Desired CI to confirm that upper and lower values change correctly (check against Steven's Excel spreadsheet)
4. Put bad characters into field, confirm that value defaults to 95 when clicking away from text box
5. Save evaluation, confirm that the desired CI value is saved in `desiredCI` of the `populationData` object
6. Reload the interpretation, confirm that the correct desired CI value is pre-filled

#### Connects to #788:

* Truncate length of returned float values to 5 decimal places (with rounding)

![image](https://cloud.githubusercontent.com/assets/4326866/16747363/ad6e2d6e-4773-11e6-94d7-69c04ce75ef5.png)

*Testing:*

1. Load variant with population tab
2. Confirm that the values in population tab do not exceed 5 decimal places
